src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVmss.md

```diff
@@ -25,7 +25,7 @@ This cmdlet takes a **VirtualMachineScaleSet** object as input.
 
 ### Example 1: Create a VMSS
 ```
-PS C:\># Common
+PS C:\> # Common
```

Comment:
It looks liek this example doesn't use prompts except on this line.  If we're going to touch an example, let's actually clean it up